### ISSUE TITLE
Boolean values misinterpreted

### ISSUE THREAD

**Issue by bajalovic** [rails/jbuilder#240]
I put in following code in `settings.jbuilder`

    json.settings @user.settings

After I added a new boolean field to `settings` model I kept getting `0` or `1` as values for this field (but other boolean fields were `true` or `false`). And this was on `production`, on my development env it was fine.

When I updated the code to 

    @settings = @user.settings # in my controller

and in jbuilder view file

    json.settings @settings 

everything is fine on both development and production env..

Is this a bug or what?

**Comment by rwz** [comment]
Could you provide steps to reproduce?

**Comment by bajalovic** [comment]
I had a model with few `boolean` and string fields (that stored user's settings data). I created a new migration and added one more `boolean` field with default value `false`. After that, on production I was getting `0` or `1` instead of `false`/`true`. 

On production we are using Passenger on nginx, and on development I have Puma running.

**Comment by rwz** [comment]
These are not steps to reproduce. Closing issue for now. Feel free to comment when you figure out steps to reproduce.

**Comment by adelgado** [comment]
@bajalovic Thanks for the very clear steps for reproduction. I was able to reproduce it. Here's the way I could solve it, for other people who end up here while looking for an answer: I've implemented, on the model I was trying to render, a method that has the same name as the property (`foo`), and returning the equivalent method with a ?  (`self.foo?`).

Basically
```
def foo
  self.foo?
end
```

I am not sure if this is the correct or cleanest approach, as I'm a newcomer to the Rails world.
